provider/backup_utils.py

```diff
@@ -405,6 +405,21 @@ def create_image_by_params(vm, params, image_name):
     return image
 
 
+def create_image_with_data_file(vm, params, image_name):
+    """Create image with data_file"""
+    image_list = []
+    image_params = params.object_params(image_name)
+    data_file_tag = image_params.get("data_file")
+    if data_file_tag:
+        data_file_image = sp_admin.volume_define_by_params(data_file_tag, params)
+        data_file_image.hotplug(vm)
+        image_list.append(data_file_image)
+    image = sp_admin.volume_define_by_params(image_name, params)
+    image.hotplug(vm)
```

Comment:
hotplug here will not only hotplug the protocol node but also hotplug the format node, refer to your manual case, we only hotplugged protocol node, so is that OK with format node hotplugged here?

Reply:
Sorry that I just missed this PR, I will review the left later.

Reply:
Case miss this step, have added it, thanks zhenchao.